salt/modules/cassandra.py

```diff
@@ -43,7 +43,7 @@ def __virtual__():
         )
 
     if HAS_PYCASSA and salt.utils.path.which("nodetool"):
-        return "cassandra"
+        return "cassandra_mod"
```

Comment:
from @s0undt3ch 's comment:  "rename salt/modules/cassandra.py to salt/modules/cassandra_mod.py, on `__virtualname__` set it to cassandra"

So we will need to rename the file and set `__virtualname__` to cassandra.

Also it looks like there are some mac/windows failures.

Reply:
i fixed this.